packages/react-native/Libraries/Image/RCTImagePlugins.h

```diff
@@ -33,7 +33,7 @@ Class RCTGIFImageDecoderCls(void) __attribute__((used));
 Class RCTImageEditingManagerCls(void) __attribute__((used));
 Class RCTImageLoaderCls(void) __attribute__((used));
 Class RCTImageStoreManagerCls(void) __attribute__((used));
-Class RCTLocalAssetImageLoaderCls(void) __attribute__((used));
+Class RCTBundleAssetImageLoaderCls(void) __attribute__((used));
```

Comment:
This file is automatically [generated by an internal script](https://github.com/facebook/react-native/pull/37232/files#diff-2f1b719a9f0e2a007a0a0d2eb6cfe34db6c40025b48419a3324c054525d26b78R7), so please, revert this change. 😅